backend/src/main/java/ai/verta/modeldb/versioning/FindRepositoriesQuery.java

```diff
@@ -282,8 +282,16 @@ private void setOwnerPredicate(
                 roleService,
                 ModelDBResourceEnum.ModelDBServiceResourceTypes.REPOSITORY,
                 userInfoList);
+        Set<String> datasetIdSet =
+            RdbmsUtils.getResourceIdsFromUserWorkspaces(
+                authService,
+                roleService,
+                ModelDBResourceEnum.ModelDBServiceResourceTypes.DATASET,
+                userInfoList);
         List<Long> resourcesIds =
             repositoryIdSet.stream().map(Long::parseLong).collect(Collectors.toList());
+        resourcesIds.addAll(
+            datasetIdSet.stream().map(Long::parseLong).collect(Collectors.toList()));
```

Comment:
I'm confused why we're adding datasets in FindRepository.

Reply:
it was irrelevant there so removed